space_age.py

```diff
@@ -0,0 +1,10 @@
+class SpaceAge(object):
+    def __init__(self, seconds):
+        planets = [
+            ['Earth', 1], ['Mercury', 0.2408467], ['Venus', 0.61519726],
+            ['Mars', 1.8808158], ['Jupiter', 11.862615], ['Saturn', 29.447498],
+            ['Uranus', 84.016846], ['Neptune', 164.79132]]
```

Comment:
@zarniak I think that the syntax checking tool might complain that the closing `]` should be in a new line ;) 

Bad:
```
foobar = [
    1, 2, 3,
    4, 5, 6]
```

Good:
```
foobar = [
    1, 2, 3,
    4, 5, 6,
]
```